### PR TITLE
Update webkit recipes to 2.32.0

### PR DESCRIPTION
**Pull request by psaavedra**

### CODE REVIEW

File: recipes-browser/wpewebkit/wpewebkit_2.32.0.bb [hunk 1]
```diff
@@ -8,13 +8,13 @@ SRC_URI = "\
...
+SRC_URI[sha256sum] = "6cfb18af9a180eeffffcaf34fea68d867ee59f633d811ced92bbead2d184b6ea"
 
 DEPENDS += " libwpe"
 RCONFLICTS_${PN} = "libwpe (< 1.4) wpebackend-fdo (< 1.6)"
 
 SRC_URI_class-devupstream = "git://git.webkit.org/WebKit.git;branch=master"
-SRCREV_class-devupstream = "8761d758f83fd7fda3d98a971cee02c38f765637"
+SRCREV_class-devupstream = "b410adfdbd2835d90a4bf86282ba742448848c94"
 
 # This is a pre-release, so de-prioritise compared to stable releases.
```
Comment (philn) [inline recipes-browser/wpewebkit/wpewebkit_2.32.0.bb:19]: Maybe remove these 2 lines? :)

**Review by philn — CHANGES_REQUESTED** [review]

File: recipes-browser/wpewebkit/wpewebkit_2.32.0.bb [hunk 2]
```diff
@@ -8,13 +8,13 @@ SRC_URI = "\
     file://274943-GStreamer-Use-imxvideoconvert_g2d-element-inside-the.patch \
```
Comment (philn) [inline recipes-browser/wpewebkit/wpewebkit_2.32.0.bb:8]: The patches should be removed too

File: .gitlab-ci/template/include/preferred-wpe-2_32.conf [hunk 3]
```diff
@@ -1,6 +1,6 @@
 PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-fdo"
 PREFERRED_PROVIDER_virtual/libwpe = "libwpe"
-PREFERRED_VERSION_cog = "0.7.%"
-PREFERRED_VERSION_wpewebkit = "2.28.%"
-PREFERRED_VERSION_wpebackend-fdo = "1.6.%"
+PREFERRED_VERSION_cog = "0.8.%"
+PREFERRED_VERSION_wpewebkit = "2.30.0%"
```
Comment (philn) [inline .gitlab-ci/template/include/preferred-wpe-2_32.conf:4]: Typo? I guess this should be 2.32

Comment (psaavedra) [reply .gitlab-ci/template/include/preferred-wpe-2_32.conf]: :facepalm:

File: .gitlab-ci/template/include/preferred-wpe-2_32.conf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-fdo"
 PREFERRED_PROVIDER_virtual/libwpe = "libwpe"
-PREFERRED_VERSION_cog = "0.7.%"
-PREFERRED_VERSION_wpewebkit = "2.28.%"
-PREFERRED_VERSION_wpebackend-fdo = "1.6.%"
+PREFERRED_VERSION_cog = "0.8.%"
+PREFERRED_VERSION_wpewebkit = "2.32.0%"
+PREFERRED_VERSION_wpebackend-fdo = "1.8.%"
```
Comment (aperezdc) [inline .gitlab-ci/template/include/preferred-wpe-2_32.conf:5]: WPEBackend-fdo 1.10.0 will be released in the coming days, maybe you want to wait until then and depend on 1.10.0 instead.

Comment (psaavedra) [reply .gitlab-ci/template/include/preferred-wpe-2_32.conf:5]: It is a possibility but having in account WPE is [compatible with 1.8](https://wpewebkit.org/release/schedule/) I don't see this as a blocker.

**Review by philn — CHANGES_REQUESTED** [review]
Please remove the orphaned patches too :)

**Comment by psaavedra** [comment]
> Please remove the orphaned patches too :)

:-m I think the that I left is still used by the 2.30 recipe.

File: recipes-browser/wpewebkit/wpewebkit_2.31.91.bb [hunk 5]
```diff
@@ -1,20 +0,0 @@
-require wpewebkit.inc
-require conf/include/devupstream.inc
-
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-SRC_URI = "\
-    https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz \
-    file://0001-EME-Fix-KeySystem-permission-handling.patch \
-    file://274943-GStreamer-Use-imxvideoconvert_g2d-element-inside-the.patch \
```
Comment (philn) [inline recipes-browser/wpewebkit/wpewebkit_2.31.91.bb:8]: I am talking about these.

Comment (psaavedra) [reply recipes-browser/wpewebkit/wpewebkit_2.31.91.bb:8]: 0001-EME-Fix-KeySystem-permission-handling.patch removed.
274943-GStreamer-Use-imxvideoconvert_g2d-element-inside-the.patch still in use by   wpewebkit_2.30.6.bb.

**Review by philn — APPROVED** [review]

**Pull request merged**